core/domain/user_services_test.py

```diff
@@ -863,6 +1122,32 @@ def test_removing_default_user_role_raises_exception(self):
         with self.assertRaisesRegex(Exception, error_msg):
             user_services.remove_user_role(user_id, feconf.ROLE_ID_FULL_USER)
 
+    def test_is_user_registered(self):
```

Comment:
Try to state more explicitly what's being tested (under what conditions should the user be registered)?

Reply:
Done